src/crate/client/http.py

```diff
@@ -128,11 +128,13 @@ def __init__(self, servers=None, timeout=None, ca_cert=None,
         self._inactive_servers = []
         self._http_timeout = timeout
         pool_kw = {}
-        if ca_cert is None:
-            ca_cert = os.environ.get("REQUESTS_CA_BUNDLE", None)
-        if ca_cert is not None:
-            pool_kw['ca_certs'] = ca_cert
-            pool_kw['cert_reqs'] = verify_ssl_cert and 'REQUIRED' or 'NONE'
+        https = not any(['http:' in uri.lower() for uri in servers])
```

Comment:
the check is actually wrong in cases where you have both http and https uris. if at least one https url exists it should match so you need to check the positive case. also you should check just the beginning of the url like this: 

```
https =  any([uri.lower().startswith('https:') for uri in servers])
```